<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prop</class>
 <widget class="QDialog" name="Prop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Properties of the user</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Departments</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Roles</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>ОК</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/images/images/accept.png</normaloff>:/images/images/accept.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QListWidget" name="lstDept"/>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QListWidget" name="lstRoles"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cencel</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/images/images/close.png</normaloff>:/images/images/close.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
